test/mpeg-encoder.test.js

```diff
@@ -0,0 +1,44 @@
21, 242, 11, 96, 128, 140, 211, 194, 200, 188, 108, 116, 51, 27, 113, 78, 107, 25, 0, 0, 0, 64, 172, 165, 0, 36, 7, 24, 198, 49, 141, 209, 17, 17, 17, 17, 221, 221, 221, 221, 209, 17, 17, 17, 17, 221, 221, 221, 221, 209, 17, 17, 17, 17, 221, 221, 221, 221, 209, 17, 17, 19, 255, 255, 243, 253, 221, 221, 209, 17, 17, 17, 4, 1, 129, 147, 38, 76, 153, 50, 100, 200, 16, 32, 64, 129, 2, 4, 8, 19, 38, 76, 153, 50, 100, 201, 147, 8, 32, 64, 129, 0, 64, 0, 0, 32, 6, 3, 1, 128, 192, 96, 48, 25, 50, 97, 4, 8, 16, 32, 64, 129, 2, 4, 201, 147, 38, 76, 153, 50, 100, 200, 16, 32, 64, 129, 2, 4, 16, 78, 238, 238, 238, 238, 34, 34, 35, 255, 255, 255, 196, 69, 221, 221, 221, 158, 153, 2, 4, 8, 16, 32, 64, 129, 2, 100, 201, 147, 38, 76, 153, 50, 97, 4, 8, 16, 4, 0, 0, 0, 1, 0, 48, 24, 12, 6, 3, 1, 133, 147, 216, 146, 72, 146, 147, 204, 0, 1, 21, 85, 85, 85, 85, 153, 153, 153, 153, 149, 85, 85, 85, 85, 153, 153, 153, 155, 213, 85, 125, 153, 85, 85, 85, 84, 40, 106, 170, 170, 170, 169, 153, 153, 153, 153, 170, 170, 170, 163, 137, 26, 68, 137, 18, 36, 72, 145, 34, 72, 145, 34, 68, 137, 18, 36, 72, 137, 18, 36, 72, 145, 34, 68, 137, 34, 68, 137, 18, 36, 72, 145, 35, 72, 145, 34, 68, 137, 20, 77, 170, 170, 170, 170, 169, 153, 153, 153, 153, 170, 170, 175, 255, 255, 255, 85, 85, 51, 51, 50, 68, 137, 26, 170, 162, 68, 137, 18, 36, 68, 137, 18, 36, 72, 145, 34, 68, 145, 34, 68, 137, 18, 36, 72, 24, 137, 18, 36, 72, 145, 34, 68, 20, 145, 34, 68, 137, 18, 36, 72, 24, 137, 18, 36, 72, 145, 34, 68, 137, 34, 68, 137, 18, 36, 72, 145, 34, 36, 72, 145, 34, 68, 136, 40, 80, 80, 80, 80, 72, 40, 42, 98, 10, 106, 40, 24, 0, 72, 64, 0, 32, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 255, 251, 146, 196, 0, 3, 192, 0, 1, 164, 0, 0, 0, 32, 0, 0, 52, 128, 0, 0, 4, 76, 65, 77, 69, 3, 0, 9, 8, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]])
```

Comment:
You can’t just disable line because of the length.

Split it to multiple lines:

```js
expect(messages).toEqual([[
  255, 251, 144, 196, 0, 0, 21, 242, 11, 96, 128, 140, 
  211, 194, 200, 188, 108, 116, 51, 27, 113, 78
]])
```

Reply:
done